test/controllers/journeys/expenses/GoodsToSellOrUseControllerSpec.scala

```diff
@@ -43,118 +47,218 @@ class GoodsToSellOrUseControllerSpec extends SpecBase with MockitoSugar {
   lazy val goodsToSellOrUseRoute = GoodsToSellOrUseController.onPageLoad(NormalMode).url
 
   val formProvider = new GoodsToSellOrUseFormProvider()
-  val form         = formProvider()
+  val businessId   = "SJPR05893938418"
```

Comment:
can now replace all businessId =  "SJPR05893938418" with 'stubbedBusinessId' in the BaseSpec